cmake/HPX_AddConfigTest.cmake

```diff
@@ -57,14 +57,14 @@ macro(add_hpx_config_test variable)
             "-DCOMPILE_DEFINITIONS=${CONFIG_TEST_COMPILE_DEFINITIONS}"
           RUN_OUTPUT_VARIABLE ${variable}_OUTPUT
           ARGS ${${variable}_ARGS})
-          if(${variable}_COMPILE_RESULT AND NOT ${variable}_RUN_RESULT)
-            set(${variable}_RESULT TRUE)
-          else()
-            set(${variable}_RESULT FALSE)
```

Comment:
Actually, this is just a formatting change (indentation) which in the diff looks like a real change. So there is no need to revert anything.